common/constants.go

```diff
@@ -116,6 +118,10 @@ const (
 	// BatcherLocalDomainName is domain name for batcher workflows running in local cluster
 	// Batcher cannot use SystemLocalDomain because auth
 	BatcherLocalDomainName = "cadence-batcher"
+	// ShadowerDomainID is domain id for workflow shadower local domain
+	ShadowerDomainID = "59c51119-1b41-4a28-986d-d6e377716f82"
```

Comment:
Why do we need the domain ID?

Reply:
Worker will talk to metadata store directly for registering system domain instead of calling frontend. so it needs both the domain name and ID.

Reply:
So we also need to add this domain register in the DB setup script?

Reply:
I don't think so. Worker will ensure the domain exists on start up and register the domain if not. I also don't see anything related to `cadence-batcher` in DB setup scripts.

Reply:
Will we have the local shadower domain per region?

Reply:
Yes. It's ok for local domains to have the same domainID in different regions, all system domains follows this pattern.